user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/GQI_Extensions_Logging.md

```diff
@@ -0,0 +1,138 @@
+---
+uid: GQI_Extensions_Logging
+---
+
+# Logging
+
+From DataMiner 10.4.5/10.5.0 onwards<!-- RN 39043 -->, [ad hoc data sources](xref:Configuring_an_ad_hoc_data_source_in_a_query) and [custom operators](xref:GQI_Custom_Operator) can log their own messages and exceptions within GQI, complementing the existing [GQI core logging functionality](xref:GQI_Logging) (available from DataMiner 10.4.0/10.4.4).
+
+## Prerequisites
```

Comment:
I still like the original section title better here. Or you could replace it with your "Logging messages" as well.
Either way, this section contains the main steps I wanted to explain and not prerequisites.

```suggestion
## How to log
```